c7n/resources/s3.py

```diff
@@ -161,6 +161,11 @@ def assemble_bucket(item):
                 methods.append((m, k, default, select))
                 continue
             else:
+                if e.response['Error']['Code'] == 'AccessDenied':
+                    if 'DeniedActions' in b:
+                        b['DeniedActions'].append(m)
```

Comment:
re annotation key, we try to keep custodian added fields under the `c7n:` prefix.. ie `c7n:DeniedActions`